INSTALL.md

```diff
@@ -4,30 +4,21 @@ Install
  - `gcc` 4.4 - 4.8 (depends on current CUDA version)
   - *Debian/Ubuntu:* `sudo apt-get install gcc-4.4 build-essential`
   - *Arch Linux:* `sudo pacman -S base-devel`
- - [CUDA 5.0](https://developer.nvidia.com/cuda-downloads) or higher
-  - *Debian/Ubuntu:* `sudo apt-get install nvidia-common nvidia-current nvidia-cuda-toolkit nvidia-cuda-dev`
-  - *Arch Linux:* `sudo pacman -S cuda`
- - one Nvidia **CUDA** compatible **GPU** with compute capability >= 2.0
-  - [full list](https://developer.nvidia.com/cuda-gpus) of CUDA GPUs and their *compute capability*
- - `boost` >= 1.48
-   - compile time headers
-   - `boost::program_options`
+ - `alpaka`
+  - included as git submodule
+ - `boost` >= 1.65.1
+   - dependency of alpaka
    - *Debian/Ubuntu:* `sudo apt-get install libboost-dev libboost-program-options-dev`
    - *Arch Linux:* `sudo pacman -S boost`
    - or download from [http://www.boost.org/](http://sourceforge.net/projects/boost/files/boost/1.55.0/boost_1_55_0.tar.gz/download)
- - `CMake` >= 2.8.12.2
+ - `CMake` >= 3.8
```

Comment:
THis should be increased directly to `3.15` because mallocMC depends on alpaka and alpaka enforces `3.15`

Reply:
done